### PR TITLE
Responsive breakpoints demo

### PR DESCRIPTION
**Pull request by LB0ge**
Asana task link: https://app.asana.com/0/0/1204724384298609/f

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]
LGTM!

File: samples/dashboards/responsive/responsive-breakpoints/demo.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                        ]
+                    }
+                ]
+            }
+        ]
+    },
+    responsiveBreakpoints: {
+        small: 300,
+        medium: 500,
+        large: 600
```
Comment (bre1470) [inline samples/dashboards/responsive/responsive-breakpoints/demo.js:44]: I do not see an effect:
Screenshot 2023-06-28 at 14 59 19

Comment (karolkolodziej) [reply samples/dashboards/responsive/responsive-breakpoints/demo.js:44]: You need to enable the edit mode and click one of the responsive buttons, perhaps we should describe that?

Comment (bre1470) [reply samples/dashboards/responsive/responsive-breakpoints/demo.js:44]: Oh! I agree. A description in source or details would be helpful.

**Pull request merged**